base/domain-sot.yml

```diff
@@ -0,0 +1,380 @@
+          TSE DING TSE NGATA 📄 - ho bala melawana le dipehelo_
+
+          (kapa Araba MENU 📌 _ ho kgutlela ho menu wa pele_)
+
+  utter_more_terms:
+      - text: |
+          O netefatsa hore wa ikarabella bakeng sa tlhokomelo ya hao ya bophelo bo botle le kalafo. Tshebeletso ena e fana ka lesedi feela.
+
+          Tshebeletso ena ha e nke sebaka sa dikeletso/tlhahlobo/kalafo tsa ngaka. Fumana dikeletso tsa mofani wa ditshebeletso tsa bophelo bo botle mabapi le boemo ba hao ba kalafo/tlhokomelo, haholo-holo haeba o ba le matshwao a matla.
+
+          O netefatsa hore ha wa lokela ho se tsotelle/ ho dieha ho batla boeletsi ba ngaka mabapi le kalafo/tlhokomelo ka lebaka la tshebeletso ena. Kotsi ya ho itshetleha leseding lena e boikarabellong ba hao.
+
+          Araba ka le leng la mantswe a ngotsweng ka *BOTENYA* ho tswela pele
+          AMOHELA ✅ - _ho amohela melawana le dipehelo_
+          (kapa Araba MENU 📌 _ ho kgutlela ho menu wa pele_)
+
+  utter_ask_province:
+      - text: |
+          O dula profenseng efe hona tje?
+
+          Araba:
+          *1.* EASTERN CAPE
+          *2.* FREE STATE
+          *3.* GAUTENG
+          *4.* KWAZULU NATAL
+          *5.* LIMPOPO
+          *6.* MPUMALANGA
+          *7.* NORTH WEST
+          *8.* NORTHERN CAPE
+          *9.* WESTERN CAPE
+
+  utter_incorrect_selection:
+    - text: |
+        Tshebeletso ena e sebetsa hantle ha o sebedisa kgetho tse ngotsweng ka *BOTENYA*.
+        📌Araba *MENU* ho kgutlela ho menu wa pele
+
+  utter_incorrect_location:
+    - text: |
+        If you have typed your address incorrectly, please try again. If you are unable to provide your address, please TYPE the name of your Suburb, Township, Town or Village (or nearest)
+
+  utter_ask_age:
+    - text: |
+        O na le dilemo tse kae?
+
+        *1.* <18
+        *2.* 18-39
+        *3.* 40-65
+        *4.* >65
+
+  utter_ask_gender:
+    - text: |
+        Ka kopo fana ka bong ba hao:
+
+        Araba:
+        *1.* MONNA
+        *2.* MOSADI
+        *3.* E NNGWE
+        *4.* KE KGETHA HO SE TJHO
+
+  utter_ask_location:
+    - text: |
+        Re hloka ho fumana sebaka seo o leng ho sona.
+
+        Ka kopo fetisa sebaka seo o leng ho sona 📍 ka WhatsApp ka ho latela mehato ena:
+        - Penya setshwantsho sa attach (+ kapa 📎) tlaase ho WhatsApp.
+        - Kgetha sebaka
+        - Sena se tla tlisa mmapa o nang le pin e bontshang sebaka seo o leng ho sona hona tjena. Penya *Romella Sebaka seo O leng ho Sona* haeba e le aterese ya hao.
+        - O ka kgetha dibaka tse ding tse haufi haeba o nahana hore GPS e fositse.
+
+        Kapa THAEPA lebitso la Surburb, Lekeishine, Toropo kapa Motse (kapa haufi le moo)?
+
+  utter_ask_location_confirm:
+    - text: |
+        ⬛⬛⬛⬜⬜⬜⬜⬜⬜⬜⬜
+
+        Re sebedisa Google Location Services ho netefatsa dibaka. Ka kopo netefatsa hore aterese e ka tlase e nepahetse ho latela dintlha tseo o faneng ka tsona:
+        {location}
```

Comment:
This location needs to be surrounded by the `----------------------------` characters